gravitee-apim-portal-webui-next/src/app/api/api-details/api-tab-details/api-tab-details.component.html

```diff
@@ -19,38 +19,36 @@
   <div class="api-details__tab-details-content">
     @if (homepageData$ | async; as homepageData) {
       @if (homepageData.result) {
-        <app-page [page]="homepageData.result" [pages]="pages" [apiId]="api.id" />
+        <app-page [page]="homepageData.result" [pages]="pages" [apiId]="api.id"/>
       } @else {
-        <div class="m3-body-large" i18n="@@apiDetailsNotAvailable">No details available</div>
-        <div class="m3-body-medium" i18n="@@apiDetailsNotAvailableExplanation">
-          It seems that there is currently no details available for this API. If you’re seeking information, feel free to explore the
+        <div class="m3-body-large">No details available</div>
+        <div class="m3-body-medium">
+          It seems that there is currently no details available for this API. If you’re seeking information, feel free
+          to explore the
           documentation available.
         </div>
       }
     } @else {
-      <app-loader />
+      <app-loader/>
     }
   </div>
 
   <div>
     <mat-card appearance="raised" class="api-details__tab-details-info__card">
```

Comment:
If product decides that they want all fields to be dynamic (aka not displaying any hard-coded values), then in the code, we should only show the sidebar if there are elements in the `apiInformation` object. For the moment, the `mat-card` that houses the sidebar is always shown regardless if the api headers list is empty.

Reply:
Done